### PR TITLE
#752 Add e2e tests to verify that users can see the correct relation between connectors in the pipeline graph

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #752 )
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/cypress/integration/pipelineNewPage.test.js [hunk 1]
```diff
@@ -143,4 +143,123 @@ describe('PipelineNewPage', () => {
       .get('.node-type')
       .should('not.be.exist');
   });
+
+  context.only('Graph connectors', () => {
```
Comment (konekoya) [inline ohara-manager/client/cypress/integration/pipelineNewPage.test.js]: Please remove the `.only`...

Comment (konekoya) [reply ohara-manager/client/cypress/integration/pipelineNewPage.test.js]: Maybe use a more descriptive name for the test: Test connector graph relation?

**Comment by konekoya** [comment]
@wu87988622  Hmm, the tests are failing on my local

File: ohara-manager/client/cypress/integration/pipelineNewPage.test.js (hunk 1 above)
Comment (wu87988622) [reply ohara-manager/client/cypress/integration/pipelineNewPage.test.js]: ok

**Comment by wu87988622** [comment]
your try new commit it time out , I add the wait connector response

**Comment by konekoya** [comment]
@wu87988622 Looks good to me. Thanks for the PR :)

**Pull request merged**